3/3scale-istio-adapter/Dockerfiles/build.sh

```diff
@@ -0,0 +1,75 @@
+# ----------------------------------------------------------------------------
+# Package       : 3scale-istio-adapter 
+# Version       : code-cleanups
+# Source repo   : https://github.com/3scale/3scale-istio-adapter
+# Tested on     : RHEL_8.2
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Krishna Harsha Voora
+#
+# Disclaimer: This script has been tested in non-root mode on given platform using
+#             the mentioned version of the package. It may not work as expected 
+#             with newer versions of the package and/or distribution.
+#             In such case, please contact "Maintainer" of this script.
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+if [ -z $1 ] || [ "$1" == "laststablerelease" ]
```

Comment:
I don't get the point of the $1 == laststablerelease  ?  It looks like if someone specifies that word they get that stable old branch, maybe the one you worked on?  I think you are trying to indicate the most recently tested, which is good, but we should probably standardize on that name.  I guess the name of that branch (code-cleanups?) bugs me a little since it is a pre-release version, and ideally this would get updated to the next release which support Power.  Has "laststablerelease" been used anywhere else?

Reply:
Maybe 'lasttestedrelease' can be used which indicates the last release/branch we tested at the time the script was written.
And this will be the default behavior. A specific release can be built by explicitly passing it as a build argument. 
README provides a few script usage examples.